### ISSUE TITLE
can raptorjit allocate more than 2GB objects (strings, tables, or ffi.new object) ?

### ISSUE THREAD

**Issue by Fedora7** [raptorjit/raptorjit#205]
Is raptorjit using 64 bit pointers ? Can it allocate more than 2GB objects (strings, tables, or ffi.new object)  on x64 ? Thanks.

**Comment by lukego** [comment]
Yep. RaptorJIT always runs in "GC64" mode with full 64-bit pointers for Lua objects. This is equivalent to building LuaJIT v2.1 with `LJ_GC64=1`. This has been working well and stable for a year or more.
